.github/workflows/main.yml

```diff
@@ -41,21 +41,15 @@ jobs:
         with:
           nuget-version: '5.x'
         if: matrix.tsv.dotnet
-      - name: GitHub Actions job_id parser
-        uses: Tiryoh/gha-jobid-action@v0.1.2
-        id: job_info
-        with:
-          github_token: ${{github.token}}
-          job_name: ${{ format('{0} ({1}, {2}, {3})', github.job, matrix.tsv.target, matrix.tsv.implementation, matrix.tsv.dotnet) }}
       - name: prepare
         run: |
           git clone https://github.com/kaitai-io/kaitai_struct_tests tests
           ln -s ../compiled tests/compiled
           ./prepare-$TARGET
       - name: run
         env:
-          GH_JOB_ID: ${{steps.job_info.outputs.job_id}}
-          GH_HTML_URL: ${{steps.job_info.outputs.html_url}}
+          GH_JOB_ID: ${GITHUB_RUN_ID}
+          GH_HTML_URL: ${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}/actions/runs/${GITHUB_RUN_ID}
```

Comment:
@GreyCat Unfortunately, I'm pretty sure that `GITHUB_RUN_ID` does not give us the "job id" we want and the URL you assemble like this won't work ("run id" != "job id" in GitHub terminology).

If it was that easy, there would be really no need for https://github.com/Tiryoh/gha-jobid-action, but last I checked GitHub still doesn't provide the job id, so we need something like https://github.com/Tiryoh/gha-jobid-action if we want it.

I saw that it doesn't work on Windows because it's a Docker action, that sucks, but the only options I see are to 1) not provide the job ID for Windows actions, 2) convert https://github.com/Tiryoh/gha-jobid-action to a JavaScript action.

Reply:
Oh, I see the issue now, thanks for detailed explanation. Yeah, looks like if we ever get to port Appveyor to GHA (at this point I'm not 100% convinced that it's worth it — in the end of the day, we trade one proprietary and fragile ecosystem to another one — also proprietary and fragile), then at first we can start without job_id (that will still allow us to have at least run_id level URL, which is better than nothing), and then we can indeed port  https://github.com/Tiryoh/gha-jobid-action to JavaScript.